es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AnimatedBuilder-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Draggable",
+    "description": "Un widget que se puede arrastrar desde un DragTarget. Cuando un widget arrastrable reconoce el inicio de un gesto de arrastre, muestra un widget de retroalimentación que rastrea el dedo del usuario a través de la pantalla. Si el usuario levanta el dedo mientras está en la parte superior de un DragTarget, se le da la oportunidad de aceptar los datos transportados por el objetivo arrastrable.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Draggable-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "LongPressDraggable",
+    "description": "Hace que su hijo sea arrastrable a partir de una pulsación larga.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/LongPressDraggable-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "GestureDetector",
+    "description": "Un widget que detecta gestos. Intenta reconocer gestos que corresponden a sus llamadas de retorno no nulas. Si este widget tiene un hijo, se remite a ese hijo por su comportamiento de tamaño. Si no tiene un hijo, crece para adaptarse a los padres.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/GestureDetector-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "DragTarget",
+    "description": "Un widget que recibe datos cuando se suelta un widget arrastrable. Cuando se arrastra un arrastrable encima de un objetivo de arrastre, se pregunta al objetivo de arrastre si aceptará los datos que lleva el arrastrable. Si el usuario suelta el arrastrable encima del objetivo de arrastre (y el objetivo de arrastre ha indicado que aceptará los datos del arrastrable), se le pide al objetivo de arrastre que acepte los datos del arrastrable.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/DragTarget-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Dismissible",
+    "description": "Un widget que puede ser eliminado arrastrando en la dirección indicada. Arrastrando o lanzando este widget en la dirección DismissDirection hace que el hijo se deslice fuera de la vista. Después de que deslice la animación, si resizeDuration no es nulo, el widget Dismissible anima su altura (o anchura, la que sea perpendicular a la dirección de salida) a cero sobre resizeDuration.",
```

Comment:
A widget that can be dismissed by dragging in the indicated direction. Dragging or flinging this widget in the DismissDirection causes the child to slide out of view. Following the slide animation, if resizeDuration is non-null, the Dismissible widget animates its height (or width, whichever is perpendicular to the dismiss direction) to zero over the resizeDuration.

Un widget que puede ser descartado arrastrándolo en la dirección indicada. Arrastran o arrojando este widget en la DismissDirection causa que su hijo se deslize fuera de la vista. Seguido de la animación slide. si resizeDuration no es nulo, el widget Dismissible anima su altura( o anchura, cualquiera que sea perpendicular a DismissDirection) hasta cero durante la resizeDuration.